### PR TITLE
[AUTO] Change master to main

### PR DESCRIPTION
**Pull request by abelsiqueira**
These changes were made automatically, please check before merging.

### CODE REVIEW

File: Artifacts.toml [hunk 1]
```diff
@@ -45662,21 +45662,21 @@ lazy = true
     sha256 = "06cd7a625b6f26a1b6e0d8d714dfc57feea7bdaf60a3e188180dddc3906ecb53"
     url = "https://sparse.tamu.edu/RB/Watson/Baumann.tar.gz"
 
-["Watson/chem_master1.MM"]
+["Watson/chem_main1.MM"]
```
Comment (abelsiqueira) [inline Artifacts.toml:45665]:
```suggestion
["Watson/chem_master1.MM"]
```

File: Artifacts.toml [hunk 2]
```diff
@@ -45662,21 +45662,21 @@ lazy = true
...
 git-tree-sha1 = "0bb4f772b7e8aaa5e98243abfa075414d8bdf850"
 lazy = true
 
-    [["Watson/chem_master1.MM".download]]
+    [["Watson/chem_main1.MM".download]]
```
Comment (abelsiqueira) [inline Artifacts.toml:45669]:
```suggestion
    [["Watson/chem_master1.MM".download]]
```

File: Artifacts.toml [hunk 3]
```diff
@@ -45662,21 +45662,21 @@ lazy = true
...
     sha256 = "43e03a61e2b61e86835ed40d824d0060186447153ecc2ad9a74e8a26aa053b93"
-    url = "https://sparse.tamu.edu/MM/Watson/chem_master1.tar.gz"
+    url = "https://sparse.tamu.edu/MM/Watson/chem_main1.tar.gz"
```
Comment (abelsiqueira) [inline Artifacts.toml:45671]:
```suggestion
    url = "https://sparse.tamu.edu/MM/Watson/chem_master1.tar.gz"
```

File: Artifacts.toml [hunk 4]
```diff
@@ -45662,21 +45662,21 @@ lazy = true
...
 
-["Watson/chem_master1.RB"]
+["Watson/chem_main1.RB"]
```
Comment (abelsiqueira) [inline Artifacts.toml:45673]:
```suggestion
["Watson/chem_master1.RB"]
```

File: Artifacts.toml [hunk 5]
```diff
@@ -45662,21 +45662,21 @@ lazy = true
...
 git-tree-sha1 = "ad1ba212eb201dab08d44519e8604ae10bd7c8d3"
 lazy = true
 
-    [["Watson/chem_master1.RB".download]]
+    [["Watson/chem_main1.RB".download]]
```
Comment (abelsiqueira) [inline Artifacts.toml:45677]:
```suggestion
    [["Watson/chem_master1.RB".download]]
```

File: Artifacts.toml [hunk 6]
```diff
@@ -45662,21 +45662,21 @@ lazy = true
...
     sha256 = "bf96cea580f11d5b314ffe283900f75a59b1216e5f47e9921049abea6f1f7f51"
-    url = "https://sparse.tamu.edu/RB/Watson/chem_master1.tar.gz"
+    url = "https://sparse.tamu.edu/RB/Watson/chem_main1.tar.gz"
```
Comment (abelsiqueira) [inline Artifacts.toml:45679]:
```suggestion
    url = "https://sparse.tamu.edu/RB/Watson/chem_master1.tar.gz"
```

**Comment by dpo** [comment]
Please don’t do that

**Comment by abelsiqueira** [comment]
I changed this manually. Which part is "that"?

**Comment by amontoison** [comment]
Thanks Abel !

**Pull request merged**